Lib/unittest/case.py

```diff
@@ -1238,6 +1238,11 @@ def assertCountEqual(self, first, second, msg=None):
             msg = self._formatMessage(msg, standardMsg)
             self.fail(msg)
 
+    def addTrailingNewLine(self, line):
+        if line != '' and (line[-1] != '\n' or line[-1] != ' '):
+            line = line + '\n'
```

Comment:
Thanks for the PR. `line` is a list here and this causes test failures due to `TypeError`. `lines` seems to be a better name since it makes me think it's a single line but it's actually a list of lines.